demo/src/main/AndroidManifest.xml

```diff
@@ -12,43 +13,45 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-
         <activity android:name=".DemoConfigActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
-            </intent-filter><!-- Accept chrome links -->
-            <intent-filter>
-                <action android:name="android.intent.action.VIEW" />
-
-                <category android:name="android.intent.category.DEFAULT" />
-                <category android:name="android.intent.category.BROWSABLE" />
-
-                <data
-                    android:host="midtrans"
-                    android:scheme="demo" />
-            </intent-filter>
+            </intent-filter> <!-- Accept chrome links -->
         </activity>
         <activity android:name=".DemoProductPageActivity" />
         <activity
             android:name=".DemoOrderReviewActivity"
             android:windowSoftInputMode="stateHidden|adjustPan" />
         <activity
             android:name=".UserAddressActivity"
-            android:windowSoftInputMode="stateHidden|adjustPan">
-
-        </activity>
+            android:windowSoftInputMode="stateHidden|adjustPan"></activity>
```

Comment:
Better to close directly if there's no item inside activity like `xxxx|adjustPan"/>`